schema/spacewalk/upgrade/susemanager-schema-4.0.6-to-susemanager-schema-4.0.7/001-rhnXccdfIdent-ident.sql.oracle

```diff
@@ -0,0 +1 @@
+alter table rhnXccdfIdent modify identifier varchar2(255);
```

Comment:
Uyuni does not support Oracle (and neither SUSE Manager 4.0, which is using Uyuni as upstream).

In theory you could just generate a single .sql file. But I see that other people is still porting the stuff, so I will let you choose what to do in this case (we don't have a rule yet :-)